script/run_unit_test.sh

```diff
@@ -4,6 +4,10 @@ for file in target/debug/*-*[^\.d]; do
 	if [[ -x "$file" ]]
 	then
 		filename=$(basename $file)
+		if [[ "plume_macro-c53c304b7b109570" =~ ^plume_macro ]]; then
```

Comment:
Isn't this always true? (maybe not, I'm really bad with Bash)

Reply:
oops I tested with a value, and forgot to use the variable after :grimacing: